### PR TITLE
Add circuits for a6 release

### PR DESCRIPTION
**Pull request by smtmfft**

### CODE REVIEW

File: testool/src/main.rs [hunk 1]
```diff
@@ -21,7 +17,7 @@ use strum::EnumString;
 const REPORT_FOLDER: &str = "report";
 const CODEHASH_FILE: &str = "./codehash.txt";
 
-#[macro_use]
+// #[macro_use]
```
Comment (Brechtpd) [inline testool/src/main.rs:20]: This change is required?

File: integration-tests/src/integration_public_data_circuits.rs [hunk 2]
```diff
@@ -0,0 +1,468 @@
...
+        let protocal_address = Address::from_str(PROTOCOL_ADDRESS).unwrap();
+        block
+            .transactions
+            .iter()
+            .filter(|tx| {
+                tx.to
+                    .map(|to| {
+                        to == protocal_address
+                            && tx.input.len() > 4
+                            && tx.input[0..4] == [0xef, 0x16, 0xe8, 0x45]
```
[the remark this answers is not in the document]
Comment (smtmfft) [reply integration-tests/src/integration_public_data_circuits.rs:41]: Done

File: mock/src/anchor.rs [hunk 3]
```diff
@@ -88,7 +88,7 @@ pub fn sign(tx: &mut MockTransaction) {
 /// gen anchor call
 // anchor(l1_hash,signal_root,l1_height,parent_gas_used)
 pub fn anchor_call() -> Bytes {
-    const METHOD_SIGNATURE: u32 = 0x3d384a4b;
+    const METHOD_SIGNATURE: u32 = 0xda69d3db;
```
[the remark this answers is not in the document]
Comment (smtmfft) [reply mock/src/anchor.rs:91]: Yes, I thought put it in mock lib then make it single copy, but seems inappropriate syntax-wise, let me find a good place for it.

File: testool/src/main.rs (hunk 1 above)
Comment (smtmfft) [reply testool/src/main.rs:20]: A warning of unused import, oh a better way is to remove the whole use sentence, done.

**Pull request merged**